diff --git a/src/feature.h b/src/feature.h
index 9564f8e10..71882a98b 100644
--- a/src/feature.h
+++ b/src/feature.h
@@ -152,7 +152,7 @@
  *			keyboard in a special language mode, e.g. for typing
  *			greek.
  */
-#ifdef FEAT_BIG
+#ifdef FEAT_HUGE
 # define FEAT_LANGMAP
 #endif
 
@@ -160,7 +160,7 @@
  * +keymap		'keymap' option.  Allows you to map typed keys in
  *			Insert mode for a special language.
  */
-#ifdef FEAT_BIG
+#ifdef FEAT_HUGE
 # define FEAT_KEYMAP
 #endif
 
@@ -244,7 +244,7 @@
  * +emacs_tags		When FEAT_EMACS_TAGS defined: Include support for
  *			emacs style TAGS file.
  */
-#ifdef FEAT_BIG
+#ifdef FEAT_HUGE
 # define FEAT_EMACS_TAGS
 #endif
 
@@ -321,7 +321,7 @@
  * +printer		":hardcopy" command
  * +postscript		Printing uses PostScript file output.
  */
-#if defined(FEAT_NORMAL) && (defined(MSWIN) || defined(FEAT_EVAL)) \
+#if defined(FEAT_HUGE) && (defined(MSWIN) || defined(FEAT_EVAL)) \
 	&& !defined(AMIGA)
 # define FEAT_PRINTER
 #endif
@@ -475,7 +475,7 @@
  * +gettext		Message translations (requires +multi_lang)
  *			(only when "lang" archive unpacked)
  */
-#ifdef FEAT_NORMAL
+#ifdef FEAT_HUGE
 # define FEAT_MULTI_LANG
 #endif
 #if defined(HAVE_GETTEXT) && defined(FEAT_MULTI_LANG) \
@@ -685,7 +685,7 @@
 /*
  * +vartabs		'vartabstop' and 'varsofttabstop' options.
  */
-#ifdef FEAT_BIG
+#ifdef FEAT_HUGE
 # define FEAT_VARTABS
 #endif
 
@@ -926,13 +926,13 @@
 // Amiga console has no mouse support
 #if defined(UNIX) || defined(VMS)
 # define FEAT_MOUSE_XTERM
-# ifdef FEAT_BIG
+# ifdef FEAT_HUGE
 #  define FEAT_MOUSE_NET
 # endif
-# ifdef FEAT_BIG
+# ifdef FEAT_HUGE
 #  define FEAT_MOUSE_DEC
 # endif
-# ifdef FEAT_BIG
+# ifdef FEAT_HUGE
 #  define FEAT_MOUSE_URXVT
 # endif
 #endif
@@ -950,7 +950,7 @@
  * FEAT_MOUSE_JSB
  * FEAT_MOUSE_PTERM
  */
-#if defined(FEAT_NORMAL) && defined(HAVE_GPM)
+#if defined(FEAT_HUGE) && defined(HAVE_GPM)
 # define FEAT_MOUSE_GPM
 /*
  * +mouse_gpm/dyn   Load libgpm dynamically.
@@ -960,7 +960,7 @@
 # endif
 #endif
 
-#if defined(FEAT_NORMAL) && defined(HAVE_SYSMOUSE)
+#if defined(FEAT_HUGE) && defined(HAVE_SYSMOUSE)
 # define FEAT_SYSMOUSE
 #endif
 
@@ -1046,7 +1046,7 @@
  * cursor shape		Adjust the shape of the cursor to the mode.
  * mouse shape		Adjust the shape of the mouse pointer to the mode.
  */
-#ifdef FEAT_NORMAL
+#ifdef FEAT_HUGE
 // Win32 console can change cursor shape
 # if defined(MSWIN) && (!defined(FEAT_GUI_MSWIN) || defined(VIMDLL))
 #  define MCH_CURSOR_SHAPE
@@ -1224,7 +1224,7 @@
 /*
  * +autochdir		'autochdir' option.
  */
-#if defined(FEAT_NETBEANS_INTG) || defined(FEAT_BIG)
+#if defined(FEAT_NETBEANS_INTG) || defined(FEAT_HUGE)
 # define FEAT_AUTOCHDIR
 #endif
 
